_layouts/analytics.html

```diff
@@ -1,10 +1,16 @@
 ---
 layout: base
 ---
-{%- assign json = site.data.analytics[page.category][page.days] -%}
+{%- assign analytics_path = page.dir | split: "/" | first -%}
```

Comment:
This isn't currently working.

![image](https://user-images.githubusercontent.com/355033/67101984-a2db4180-f1ba-11e9-84dc-27443dc885c9.png)